pylint_odoo/checkers/modules_odoo.py

```diff
@@ -282,6 +287,12 @@ class ModuleChecker(misc.WrapperModuleChecker):
         }),
     )
 
+    odoo_check_versions = {
+        'useless-attribute-non-translatable': {
+            'min_odoo_version': '8.0',
```

Comment:
This message will be emitted for modules in `8.0` version too?

Reply:
@moylop260 
Sorry, you're right.
In this case the `odoo_check_versions` not work because this check is not necessary when version is different to 8.0

Reply:
If odoo.version <= 8.0 then emit the message.
What is the correct configuration in odoo_check_versions for this case?
`max_odoo_version: '8.0'`?

Don't use a custom validation for this check